docs/ref/contrib/postgres/fields.txt

```diff
@@ -482,132 +482,14 @@ using in conjunction with lookups on
     its Python native format: dictionaries, lists, strings, numbers, booleans
     and ``None``.
 
-    .. attribute:: encoder
```

Comment:
I don't think that we should remove the current documentation because it's still valid, we only need to add `.. deprecated:: 3.0` note.